policy/rating.go

```diff
@@ -43,47 +46,65 @@ func (s *Score) Rating() ScoreRating {
 		return ScoreRating_bPlus
 	case s.Value >= 65:
 		return ScoreRating_b
-	case s.Value >= 60:
+	case s.Value >= 61:
```

Comment:
Why these scoring changes? Where are they coming from?

Do we have to keep them in sync with something else?